lenskit-eval/src/main/java/org/grouplens/lenskit/eval/traintest/SimpleEvaluator.java

```diff
@@ -51,6 +52,16 @@ public SimpleEvaluator() {
     }
 
     /**
+     * An algorithm instance constructed with a name and Lenskit configuration
+     * @param name
+     * @param config Lenskit configuration
+     *
+     */
+    public SimpleEvaluator(String name, LenskitConfiguration config) {
```

Comment:
This should be `public SimpleEvaluator addAlgorithm`, instead of just `SimpleEvaluator`; then it should work. Also, coudl we move it down with the other `addAlgorithm` methods?

Thanks! Good work.